modules/neovim/init/spellcheck.nix

```diff
@@ -25,9 +28,45 @@ in {
         A list of languages that should be used for spellchecking.
 
         To add your own language files, you may place your `spell`
-        directory in either `~/.config/nvim` or the
+        directory in either {file}`~/.config/nvim` or the
```

Comment:
It should be `~/.config/nvf`

Reply:
It should actually be $XDG_CONFIG_HOME/nvf. I had forgotten to apply this change, whoops.